backend/constants.py

```
@@ -41,18 +59,154 @@
+                len(_OPUS_TIER_AGENTS),
+            )
+        else:
+            logger.debug("No agent_variants in config, using built-in defaults")
+            _load_builtin_defaults()
+
+        # DeepInfra models are not in config.json yet, use built-in
+        _DEEPINFRA_SUPPORTED_MODELS = _BUILTIN_DEEPINFRA_SUPPORTED_MODELS
+
+        _config_loaded = True
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)